packages/shell/src/shell-routes.tsx

```diff
@@ -1,9 +1,10 @@
 import { dbLoader } from '@workspace/db-react/db-loader'
 import { queryClient } from '@workspace/db-react/db-query-client'
 import { dbSettingOptions } from '@workspace/db-react/db-setting-options'
+import i18n from '@workspace/i18n'
```

Comment:
Is there a reason this must be a default export? If not, let's not use a default export but just a named export.